VandyHacks/vaken
# Sponsor Events Association

Completed backend functionality with simple UI for assigning events to companies. 

### Added functions

- `getCompanyEvents` - get events that a company owns
- `checkIdentityForEvent` - check if a company owns an event
- `assignEventToCompany` - mutation; assign the company as the event's owner and add event to eventsOwned field

### Data changes
- added `eventsOwned: [ID!]! ` to Company
- added `owner: Company` to Event
- changed `startTimestamp` to Float in Event

### To test
Go to Manage Events, you should see a list of all events, with a dropdown. Select a company in the dropdown. Upon refresh, the dropdown should be defaulted to the company associated to the event. 

### Screenshot
![image](https://user-images.githubusercontent.com/10069974/67813574-dbf39b80-fa6f-11e9-9096-900d1cc251a0.png)



### To Do
- Make actual EventsTable 
- Add functionality to add events via table
